sdk/identity/azure-identity/src/test/java/com/azure/identity/implementation/msalextensions/CrossProgramVSTest.java

```diff
@@ -38,7 +38,7 @@
 
     @Before
     public void setup() throws Exception {
-        org.junit.Assume.assumeTrue("Record".equalsIgnoreCase(System.getenv("AZURE_TEST_MODE")));
+        org.junit.Assume.assumeTrue("Skipping these tests until we mock or record it", false);
```

Comment:
Is there an issue created for this so we dont lose track of it?

Reply:
Created at https://github.com/Azure/azure-sdk-for-java/issues/5328 and added to next preview release